.github/workflows/main.yml

```diff
@@ -39,18 +41,35 @@ jobs:
       - run: tar -xvf lgbm.tar
         shell: bash
         name: Untar LGBM examples
+      - name: v1.2 hackfix
+        if: matrix.version == '1.2'
+        run: rm test/Project.toml
       - run: julia --color=yes --project -e 'using Pkg; Pkg.build(); Pkg.test(coverage=true);'
         continue-on-error: false
         env:
           LIGHTGBM_EXAMPLES_PATH: LightGBM-2.3.1
+  MLJ:
+    name: MLJ Interface enabled -- Julia ${{ matrix.version }}
+    runs-on: ubuntu-latest
```

Comment:
@yaxxie just wondering if we should add at least mac-OS as well given there are quite a few mac users of this package, ideally all 3. I think it should work with the `matrix:` and then `os:  [windows-latest, ubuntu-latest, macOS-latest]` unless you had a specific reason why you went for `runs-on`? It will add few more runs to the CI but at least we'll sort of know that the supported versions with MLJ work across platforms.

Reply:
So the short answer ia that there's no need.

The reason why is that the FFI interface is really the only thing that needs to be tested cross platform, for the rest of the code theres no reason to believe there would be platform specific behaviour. I think this is acutely true for the MLJ interface, which is a much higher level.